odbc/src/api/odbc_uri.rs

```diff
@@ -80,13 +154,8 @@ impl<'a> ODBCUri<'a> {
     pub fn remove_to_mongo_uri(&mut self) -> Result<String> {
         let user = self.remove_mandatory_attribute(USER)?;
         let pwd = self.remove_mandatory_attribute(PWD)?;
-        // TODO SQL-990: Support the PORT attribute, right now the only way to specify PORT is as
```

Comment:
Decided not to support PORT for now. We can add that later if anyone asks for it.